### PR TITLE
updating links inside framework folder to https

### PR DESCRIPTION
**Pull request by eddynaka**
- updating links from http to https (if redirects automatically)
- updating links from docs.microsoft.com/path to /path
- updating links from fwlink to real link if exists

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/dns.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ For more information about setting the element value, see [Service Identity and
   
 |Attribute|Description|  
 |---------------|-----------------|  
-|value|The DNS of the certificate. DNS is an industry-standard protocol used to locate computers on an IP-based network. Users can remember display names, such as <https://go.microsoft.com/fwlink/?prd=10929> or [https://go.microsoft.com/fwlink/?LinkID=96165](https://go.microsoft.com/fwlink/?LinkID=96165), easier than number-based addresses, such as 207.46.131.137.|  
+|value|The DNS of the certificate. DNS is an industry-standard protocol used to locate computers on an IP-based network. Users can remember display names, such as <https://go.microsoft.com/fwlink/?prd=10929> or [https://go.microsoft.com/fwlink/?LinkID=96165](https://msdn.microsoft.com/), easier than number-based addresses, such as 207.46.131.137.|
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/dns.md]: I think the link text should be changed as well.

```suggestion
|value|The DNS of the certificate. DNS is an industry-standard protocol used to locate computers on an IP-based network. Users can remember display names, such as <https://go.microsoft.com/fwlink/?prd=10929> or [https://msdn.microsoft.com/](https://msdn.microsoft.com/), easier than number-based addresses, such as 207.46.131.137.|  
```

File: docs/framework/data/adonet/downloading-sample-databases-linq-to-dataset.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The samples and walkthroughs in the LINQ to DataSet documentation use the Advent
   
 1. Open Internet Explorer.  
   
-2. Go to the [SQL Server 2005 Samples and Sample Databases](https://go.microsoft.com/fwlink/?linkid=31046) Web site.  
+2. Go to the [SQL Server 2005 Samples and Sample Databases](https://www.microsoft.com/downloads/details.aspx?FamilyId=E719ECF7-9F46-4312-AF89-6AD8702E4E6E&displaylang=en) Web site.
```
Comment (Youssef1313) [inline docs/framework/data/adonet/downloading-sample-databases-linq-to-dataset.md]:
```suggestion
2. Go to the [SQL Server 2005 Samples and Sample Databases](https://www.microsoft.com/download/details.aspx?id=10679) Web site.
```

File: docs/framework/data/adonet/downloading-sample-databases-linq-to-dataset.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ The samples and walkthroughs in the LINQ to DataSet documentation use the Advent
   
 1. Start Internet Explorer.  
   
-2. Go to the  [Microsoft SQL Server 2005 Express Edition](https://go.microsoft.com/fwlink/?LinkID=31070) download page.  
+2. Go to the  [Microsoft SQL Server 2005 Express Edition](https://www.microsoft.com/downloads/details.aspx?familyid=220549B5-0B07-4448-8848-DCC397514B41&displaylang=en) download page.
```
Comment (Youssef1313) [inline docs/framework/data/adonet/downloading-sample-databases-linq-to-dataset.md]:
```suggestion
2. Go to the  [Microsoft SQL Server 2005 Express Edition](https://www.microsoft.com/download/details.aspx?id=21844) download page.
```

File: docs/framework/data/adonet/downloading-sample-databases-linq-to-dataset.md [hunk 4]
```diff
@@ -63,7 +63,7 @@ The samples and walkthroughs in the LINQ to DataSet documentation use the Advent
   
 1. Start Internet Explorer.
```
Comment (Youssef1313) [inline docs/framework/data/adonet/downloading-sample-databases-linq-to-dataset.md]: Seems like an outdated article ?
@mairaw, Does this suggestion seem to be good ?

```suggestion
1. Start your favorite browser, such as Microsoft Edge.
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

**Comment by mairaw** [comment]
Thanks @eddynaka. It would be really helpful to our team if we can break this PR down into smaller parts. As @Youssef1313 pointed out, there might be some other changes we need to make while revisiting the links. 
Also, some FwLinks point to downloads and we don't want to remove the Fwlink in those cases. And also, as I mentioned in the other PR, we don't want to remove docs.microsoft.com from all the links, just for the ones that go to docsets with VS offline books (Windows, C++, Visual Studio, ASP.NET, NuGet, EF).

**Pull request closed without merging**